ReSharper.FSharp/src/FSharp.Psi/src/FSharp.psi

```diff
@@ -611,9 +611,12 @@ computationExpr options { stubBase="FSharpExpressionBase"; }:
 
 lambdaExpr options { stubBase="FSharpExpressionBase"; }:
   FUN
-  fSharpPattern<PAT, Patterns>*
+  lambdaParameters<LAMBDA_PARAMETERS, Parameters>
   RARROW<RARROW, RArrow>
   fSharpExpression<EXPR, Expression>;
+  
+lambdaParameters:
```

Comment:
Let's use `List` suffix? It's often used in the plugin and in other R# grammars, so it matches a bit better.
```suggestion
lambdaParametersList:
```